app/src/androidTest/java/fr/free/nrw/commons/WelcomeActivityTest.kt

```diff
@@ -34,4 +35,43 @@ class WelcomeActivityTest {
                     .check(matches(not(isDisplayed())))
         }
     }
+
+    @Test
+    fun ifSkipButtonClicked() {
+        if (!ConfigUtils.isBetaFlavour()) {
```

Comment:
I think this if statement is the wrong way round - we only want to check the skip button if we are on beta (i.e. this line needs the `!` removed).